packages/replay/src/replay.ts

```diff
@@ -426,7 +435,7 @@ export class ReplayContainer implements ReplayContainerInterface {
     this.session = session;
 
     if (!this.session.sampled) {
-      this.stop();
+      this.stop('session unsampled');
```

Comment:
No, this can happen when an error session ends - it will then be set to unsampled for the "follow up" session. Something to look into a bit more, maybe...!